confluent_kafka/__init__.py

```diff
@@ -15,10 +35,18 @@
                     OFFSET_STORED,
                     OFFSET_INVALID)
 
+__all__ = ['admin', 'AvroSerializer', 'Consumer',
+           'KafkaError', 'KafkaException',
+           'kafkatest', 'libversion', 'Message',
+           'OFFSET_BEGINNING', 'OFFSET_END', 'OFFSET_INVALID', 'OFFSET_STORED',
+           'Producer', 'DeserializingConsumer',
+           'SerializingProducer', 'TIMESTAMP_CREATE_TIME', 'TIMESTAMP_LOG_APPEND_TIME',
```

Comment:
`TIMESTAMP_CREATE_TIME` etc are enum like - it seems they would be better in an enum than as constant vlues (also we want consistency on how enums are done in this lib..).

`OFFSET_BEGINNING` et al. make more sense as constants as you have them. In .NET I actually have an `Offset` class which encapsulates these constants and the value, and this works very nicely, but I'm pretty happy with just using constants as you are here in Python.

Reply:
yeah